packages/babel/src/generation/node/parentheses.js

```diff
@@ -135,7 +135,8 @@ export function SequenceExpression(node, parent) {
     return false;
   }
 
-  if (t.isExpressionStatement(parent) && parent.expression === node) {
+  if ((t.isExpressionStatement(parent) && parent.expression === node) ||
```

Comment:
Might be clearer adding a separate `if (condition) { return false; }` for this instead of doing a `||` in the same condition.